src/component/organism/listitem/AssignCheckListItem.js

```diff
@@ -3,8 +3,7 @@ import {Text, TouchableOpacity, View} from 'react-native';
 import {APRI10, BLACK, GRAY20} from 'Root/config/color';
 import {txt} from 'Root/config/textstyle';
 import CheckBox from 'Molecules/select/CheckBox';
-import {assignCheckListItem} from 'Organism/style_organism';
-
+import {assignCheckListItem} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {assignCheckListItem} from 'Organism/style_organism copy';
```